docs/_may_milestone_.md

```diff
@@ -0,0 +1,62 @@
+## Monthly Milestone - May 2020
+
+### Overview
+
+### Goals
+
+- [ ] Migration Guide
+
+  - [ ] How to migrate LB3 components #3955 _depends on the spike result in
+        #5004_
+  - [ ] How to run LB3 app tests when the app is mounted in a LB4 project #3978
+        _depends on the spike reuslt in #4099_
+
+- [ ] Feature Parity
+
+  - [ ] Model.toObject() should preserve prototypes (e.g. Date and ObjectID
+        values) #3607
+
+- [ ] Bugs
+
+  - [ ] defining a nested array model property using @property.array decorator
+        crashes the app #4754
+
+- [ ] Docs
+
+  - [ ] [Docs] Add debug string information #4845
+  - [ ] [Docs] Add "error handling" page #4846
+
+- [ ] Miscellaneous
+
+  - [ ] How to build models, repositories and controllers dynamically at runtime
+        #4296
+  - [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+  - [ ] Integration with TypeORM #4853
```

Comment:
hmm https://github.com/strongloop/loopback-next/issues/4853 doesn't seem like a miscellaneous task 🤔 maybe move it under feature party?

Reply:
"Miscellaneous" is just a category to put anything doesn't fit in docs and bugs at this moment. if we have more integration work related, i can create a "integration" section.